src/context.rs

```diff
@@ -600,23 +600,23 @@ impl Context {
         matrix
     }
 
-    pub fn set_font_options(&self, options: FontOptions) {
+    pub fn set_font_options(&self, options: &FontOptions) {
         unsafe {
-            ffi::cairo_set_font_options(self.0, options.get_ptr())
+            ffi::cairo_set_font_options(self.0, options.to_glib_none().0)
         }
     }
 
     pub fn get_font_options(&self) -> FontOptions {
-        let out = FontOptions::new();
+        let mut out = FontOptions::new();
         unsafe {
-            ffi::cairo_get_font_options(self.0, out.get_ptr());
+            ffi::cairo_get_font_options(self.0, out.to_glib_none_mut().0);
```

Comment:
It doesn't change the internal pointer, but it does modify the underlying data structure, right? So giving it a `*mut` might be the "better written" (or "better specified") thing to do?

PS: I'm not entirely sure when `*const` vs `*mut` should be used.